__tests__/src/actions/datapackageActions.test.js

```diff
@@ -1,41 +1,46 @@
-import 'babel-polyfill'
-import configureMockStore from 'redux-mock-store'
-import nock from 'nock'
-import thunk from 'redux-thunk'
-import * as actions from '../../../src/actions/datapackageActions'
-import * as types from '../../../src/constants/actionTypes'
+import 'babel-polyfill';
+import configureMockStore from 'redux-mock-store';
+import nock from 'nock';
+import thunk from 'redux-thunk';
+import expect from "expect";
+import * as actions from '../../../src/actions/datapackageActions';
 
-const middlewares = [ thunk ]
-const mockStore = configureMockStore(middlewares)
-
-let mock2 = nock('https://dp2.com')
-              .persist()
-              .get('/')
-              .replyWithFile(200, './fixtures/dp2/datapackage.json')
+const middlewares = [ thunk ];
+const mockStore = configureMockStore(middlewares);
 
 describe('async actions', () => {
+  let timeout;
+
+  beforeAll(function() {
+    timeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
+  });
+
+  afterAll(function() {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = timeout;
+  });
 
   it('creates RECEIVE_DATAPACKAGE when fetching datapackage is done', async () => {
-    const store = mockStore({ dp: {} })
-    await store.dispatch(actions.getDataPackage('https://raw.githubusercontent.com/frictionlessdata/dpr-js/gh-pages/fixtures/dp2/datapackage.json'))
-    let receivedDp = store.getActions()
-    expect(receivedDp[1]).toBeInstanceOf(Object)
-    expect(receivedDp[1].type).toEqual('RECEIVE_DATAPACKAGE')
-    expect(receivedDp[1].dp.title).toEqual('DEMO - CBOE Volatility Index')
-    expect(receivedDp[1].dp.views[0].type).toEqual('Graph')
-    expect(receivedDp[1].dp.resources).toBeInstanceOf(Array)
-  })
+    const store = mockStore({ dp: {} });
+    await store.dispatch(actions.getDataPackage('https://raw.githubusercontent.com/frictionlessdata/dpr-js/gh-pages/fixtures/dp2/datapackage.json'));
```

Comment:
1. Will move the test or atleast mock it.
  1.1 i did not removed nock.
2. It was named like this a long back.
  2.2 that i do not know. will find it out